doc/releases/tentacle.rst

```diff
@@ -0,0 +1,395 @@
+========
+Tentacle
+========
+
+Tentacle is the 20th stable release of Ceph.
+
+v20.2.0 Tentacle
+================
+
+
+Highlights
+----------
+
+*See the sections below for more details on these items.*
+
+RADOS
+
+* FastEC: Long expected performance optimizations were added for EC pools.
+* BlueStore: Users can expect to see improved compression and a new,
+  faster WAL (write-ahead-log).
+* Data Availability Score: Users can now track a data availability score
+  for each pool in their cluster.
+
```

Comment:
Perhaps it's worth adding a point on the `omap_iterate` like: `Performance impovements for OMAP iteration`.

Reply:
Which will enhance RGW bucket listing operations, right?  If so, we should mention that here or below.

Reply:
Added.